biopet-extensions/src/main/scala/nl/lumc/sasc/biopet/extensions/bedtools/BedtoolsMerge.scala

```diff
@@ -27,15 +27,23 @@ class BedtoolsMerge(val parent: Configurable) extends Bedtools {
   @Input(doc = "Input bed file")
   var input: File = _
 
-  @Argument(doc = "Distance")
-  var dist: Option[Int] = config("dist") //default of tool is 1
+  @Argument(doc = "Distance", required = false)
+  var dist: Option[Int] = config("dist", default = 1) //default of tool is 1
 
   @Output(doc = "Output bed file")
   var output: File = _
 
+  @Argument(doc = "operation to additional columns", required = false)
+  var operation: Option[String] = None
+
+  @Argument(doc = "Additional columns to operate upon", required = false)
+  var additionalColumns: Option[List[Int]] = None
```

Comment:
is it not better to just use `List[Int]` here? When the list is empty nothing needs to be happen